content/en/blog/ospo-survey-2023.md

```diff
@@ -28,8 +28,8 @@ Please participate now; we intend to close the survey in mid-July. Privacy and c
 To take the 2023 OSPO Survey, please click the button below:
 
 * [English](https://www.research.net/r/SD6K27F)
-* [Japanese]() - Soon
-* [Chinese]() - Soon
+* [日本語](https://www.research.net/r/S95X7PD?lang=ja)
```

Comment:
```suggestion
* [日本語](https://www.research.net/r/SYL6GXD?lang=ja)
```
The CollaboTrans JP open source community has contributed to a Japanese version of the survey. This is the fixed link 🙂